stdlib/3/multiprocessing/spawn.pyi

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Dict, List, Optional
+from types import ModuleType
+
+WINEXE: bool
+WINSERVICE: bool
+# undocumented
+_python_exe: str
+
+def set_executable(executable: str) -> None: ...
+def get_executable() -> str: ...
+def is_forking(argv: List[str]) -> bool: ...
```

Comment:
Looking at the implementation, requiring a `Sequence` instead of a `List` should be enough.